packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -529,6 +529,11 @@ class GoogleMapsFlutterAndroid extends GoogleMapsFlutterPlatform {
           .toList(),
     );
 
+    if (_tileOverlays[creationId] == null) {
```

Comment:
What is the issue this change corresponds to? It's not at all clear to me what this is fixing, or why it is necessary specifically for tile overlays and no other types.